etc/copyright-exclude.txt

```diff
@@ -25,3 +25,6 @@ cli-data.zip
 .hola
 .hello
 HELIDON_THIRD_PARTY_LICENSES.xml
+/MANIFEST.MF
+/META-INF/services/
+.jar
```

Comment:
Why the `.jar` exclusion ? Are we genering JAR files outside of `target` ?

Reply:
I use jar-file for tests

Reply:
Checked-in binaries such as JAR files are a bad practice that should be avoided at all cost. AFAIK this is a Helidon JAR file that is available from Maven, why can't Maven be used here ?

Reply:
I cannot use maven for this. I put inside this jar-file a helidon-configuration-metadata.json file. This file is read during the tests.

Reply:
Why not simply hard-code the JSON file and have a way to supply the JSON file directly instead of assuming it's inside the JAR ?

Reply:
The goal of this extension and the tests related to it is to go through all the dependencies of a project and if some dependency is a helidon dependency the plugin analize it and get the information from it.

Reply:
I know that. However currently the artifact you need does not exist. You need to find another way to test this than checking-in a JAR file.

Reply:
jar removed